src/pages/Lend/Filter/FilterComponents/LendFilterMenu.vue

```diff
@@ -40,16 +40,16 @@
 				<div id="filter-section-advanced" class="filter-section" @click="showAdvancedFilters">
 					Advanced filters
 				</div>
-				<ais-state-results class="filter-section filter-show-loans-mobile-wrapper hide-for-medium">
-					<kv-button
-						class="filter-show-loans-mobile"
-						slot-scope="{ nbHits }"
-						@click.native="hideFilterMenu"
-					>
-						Show {{ nbHits }} loan{{ nbHits > 1 ? 's' : '' }}
-					</kv-button>
-				</ais-state-results>
 			</div>
+			<ais-state-results class="filter-section filter-show-loans-mobile-wrapper hide-for-medium">
+				<kv-button
+					class="filter-show-loans-mobile"
```

Comment:
The solution to the bouncing button was to remove it from the scrolling container and use a z-index/positioning to ensure its placement remains the same